src/main/flight/mixer.h

```diff
@@ -90,6 +90,11 @@ typedef struct mixerConfig_s {
     bool yaw_motors_reversed;
     uint8_t crashflip_motor_percent;
     uint8_t crashflip_expo;
+    uint16_t rpm_limiter_p;
```

Comment:
#ifdef ?

Reply:
Also must be kept because of below https://github.com/betaflight/betaflight/pull/12054#discussion_r1125645333